### ISSUE TITLE
Unable to Add Playlist keeps spinning, no way to cancel or to see what is happening (or if anything even is happening)

### ISSUE THREAD

**Issue by ylnest2018** [Threadfin/Threadfin#231]
**Describe the bug**
Infinite loop when adding the M3U playlist via URL

**Server (please complete the following information):**
- Debian 12 (Bookworm) on Proxmox
- Intel 64 Bit
- Threadfin Version and Build 1.0.26

**To Reproduce**
1. Click on playlist
2. Click on new
3. enter name and server URL - http://akontv.xyz/get.php?username=MYUSERNAME&password=MYPASSWORD&output=ts&type=m3u_plus
![image](https://github.com/Threadfin/Threadfin/assets/150770555/359aad7e-da59-4bea-a511-d1bdefcf9035)

**Expected behavior**
Assuming completion of the playlist creation (this is my first time using)

The URL I put above resolves in browsers and downloads the M3U and also works in M3U4U and other similar services

**Comment by pixel24** [comment]
By the way, the same thing happens when I try to load the playlist from a local file. The file works fine in VLC and Kodi without any issues.

2024/04/18 14:15:34 [Threadfin] Open:                   /home/threadfin/iptv.m3u
2024/04/18 14:15:34 [Threadfin] Check File:             /home/threadfin/iptv.m3u
2024/04/18 14:15:34 [Threadfin] [ERROR] Invalid M3U file, an extended M3U file is required. () - EC: 0
2024/04/18 14:15:34 [Threadfin] Download:               http://hd-max.org:8080/Y217qk7DEZ/6Sf1ZPoPd1/3463
2024/04/18 14:15:36 [Threadfin] Check File:             http://hd-max.org:8080/Y217qk7DEZ/6Sf1ZPoPd1/3463
2024/04/18 14:15:37 [Threadfin] [ERROR] Invalid M3U file, an extended M3U file is required. () - EC: 0
2024/04/18 14:15:37 [Threadfin] All streams:            0
2024/04/18 14:15:37 [Threadfin] Active streams:         0
2024/04/18 14:15:37 [Threadfin] Filter:                 0
2024/04/18 14:15:37 [Threadfin] [ERROR] writev tcp 172.18.0.2:34400->192.168.83.141:36798: writev: broken pipe (Unknown error / warning (1022)) - EC: 1022

**Comment by ylnest2018** [comment]
I worked out my issue, the setup instructions said to use
1cpu
512mb ram
2GB hdd

I went to 2 cpu 4gb ram and 2GB hdd and then instantly had success

**Comment by doofit** [comment]
I am having the same issue. Getting a dial tcp i/o timeout though.

**Comment by ylnest2018** [comment]
Ok so what fixed it for me was a few things.
1. Being patient
2. Assigning more resources than suggested in the setup guide, at least initially because until I set 2 CPUs and at least 4gb RAM it essentially kept dying trying to set up. Once set up though it uses next to no resources so I’m not sure if it’ll have gotten there after like 4hrs or something without being touched in any way but I’ll never know.
3. Being patient some more, even with all the added resources I noticed that if I watched proxmox’s resources I would see the point where threadfin was done processing whatever it was doing, because the UI seems like it is in a death spiral and nothing doing, if trying to click anything because that process finishes I think that’s where there error generates, because doing it the way I just said I realised there was no “process complete” message, but seeing that there was nothing processing and then refreshing to the main page, it was obvious it finished what it was doing.


I know these aren’t major ‘workarounds’ but worth noting because you might’ve been expecting wha to was expecting and then ended up in the same boat.
